docs/AboutUs.md

```diff
@@ -40,6 +40,16 @@ You can reach us at the email `cobb[at]comp.nus.edu.sg`
 * Role: Developer
 * Responsibilities: Data
 
+### Chng Ian
+
+<img src="images/ian.png" width="200px">
```

Comment:
> The filename of the profile photo should be docs/images/github_username_in_lower_case.png
> Note the need for lower case ( why lowercase?) e.g. JohnDoe123 -> docs/images/johndoe123.png not docs/images/JohnDoe123.png.

Should change your image file name so that they can track properly